Example/patches/react-native-box2d+0.2.5.patch

```diff
@@ -72,7 +72,17 @@ index 41726f4..0580d5d 100644
    defaultConfig {
      minSdkVersion 21
      targetSdkVersion getExtOrIntegerDefault('targetSdkVersion')
-@@ -342,8 +344,8 @@ def nativeBuildDependsOn(dependsOnTask, variant) {
+@@ -176,7 +178,8 @@ def thirdPartyVersionsFile = new File("${androidSourcesDir.toString()}/ReactAndr
+ def thirdPartyVersions = new Properties()
+ thirdPartyVersions.load(new FileInputStream(thirdPartyVersionsFile))
+ 
+-def BOOST_VERSION = thirdPartyVersions["BOOST_VERSION"]
++
```

Comment:
Let's remove this unnecessary newline (probably you'll need to re-generate the patch).

Reply:
Do we really need to do it?

Reply:
not really but doesn't sound like much effort